### DISCUSSION TITLE
Wagmi v2

### DISCUSSION THREAD

**Discussion by samwhosung** [unicape/use-wagmi#91] [Q&A]
Wagmi v2 is in beta right now, any plans to implement it?

**Comment by unicape** [comment]
[wagmi-v2](https://github.com/unicape/use-wagmi/tree/wagmi-v2) This is ongoing, but I am usually quite busy with work. It is expected to be completed some time after wagmi v2 is officially released.

**Reply by ennioVisco** [reply]
@unicape is it done now?

**Reply by unicape** [reply]
Yes, it has been published
